tools/xdptrace.wprp

```diff
@@ -135,6 +135,11 @@
                 <Keyword Value="0x7FFFFFFFFFFFFFFF" />
             </Keywords>
         </EventProvider>
+        <EventProvider Id="EP_XdpFnTestDrvWpp" Name="9DF435CB-9EEB-430B-A795-EF2B8845F7F3" Level="5">
```

Comment:
does this need `NonPagedMemory="true"` like some of the WPP traces above?

Reply:
@nigriMSFT Any idea?